README.md

```diff
@@ -6,6 +6,14 @@ Geoweaver is an in-browser software allowing users to easily compose and execute
 
 It can be run from both local and remote (distributed) machines.
 
+Why choose Geoweaver?
+1) Safely Store all your progress along the way.
+2) Visualise your results using the tools that you already use.
+3) Stay organised through out your research
+4) Supports scheduling jobs / executions to run at specific times, dates and intervals using CRON for automation tasks.
```

Comment:
I mean you need to add the full explained name of CRON. For people outside computer science, they have no idea what CRON is.